src/components/TeacherViews/LessonOverview/LessonOverview.view.tsx

```diff
@@ -222,6 +223,12 @@ export function LessonOverview({ lesson }: LessonOverviewProps) {
         showDownloadAll={showDownloadAll}
       />
       <MaxWidth $ph={16} $pb={80}>
+        <NewContentBanner
+          keyStageSlug={keyStageSlug ?? ""}
```

Comment:
in what cases is keyStageSlug undefined? what happens if the banner receives empty strings for the props?

Reply:
I believe it's just Specialist pages that don't have keyStageSlug available both canonical and pathway pages should have it available

Reply:
ok, should be safe here because the banner component has an early return if the keystage isn't in the lookup, but I'd usually prefer we handle undefined values rather than defaulting to empty strings!